packages/ui/core-components/src/lib/atoms/inputs/button-group/ButtonGroup.svelte

```diff
@@ -22,6 +22,15 @@
 	/** @type {keyof typeof presets | undefined} */
 	export let preset = undefined;
 
+	// for Tabs styling
+	export let display;
```

Comment:
`= 'buttons'` ?

Reply:
How do we feel about display = 'buttons', and a error state that tells the user when display string does not exist

(similar to how we handle when presets do not exist)
![image](https://github.com/user-attachments/assets/98cf1b13-259c-4f87-8a05-f060b0cb8049)